samigo/samigo-app/src/webapp/js/authoringQuestionCancellation.js

```diff
@@ -0,0 +1,44 @@
+// S2U-14
+// When the Cancel Question button is clicked we want to open
+// the Modal to choose which method for cancellation should be selected
+// To apply it to the correct item we need modify the onclick handler
+
+(() => {
+
+    // For B3 we still need JQuery, isolating it here
+    function openModal(modalElement) {
+        // B3 version
+        $(modalElement).modal();
+
+        // B5 version
+        // new bootstrap.Modal(modalElement).show();
```

Comment:
We can't forget fix it when merging on 24.x. :)

Reply:
Yes, right!